content/en/docs/how-tos/creating-a-pipeline.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Creating a Pipeline"
+description: How to use existing functionalities to create a simple 2-stage pipeline
+---
+
+Prow does not support pipelines. Hence, when submitting a new pull request, all tests are executed, including resource-intensive tests that can lead to increased infrastructure costs. To address this limitation, OpenShift CI introduces an abstraction layer utilizing GitHub's branch protection rules. This enables the use of both manual (triggered by a user) and automatic (triggered by a bot) two-stage pipelines.
+
+## Marking CI job as second stage job
+
+To enable two-stage pipelines, two actions are required. First, a specific branch protection setting must be activated within the Prow configuration of the repository (or organization).
+
+{{< alert title="Note" color="info" >}}
+Branch protection configuration could be also enabled on branch level. It will however disable possibility of using automatic pipelines as they work on repository and organization level.
+{{< /alert >}}
+
+```yaml
+branch-protection:
+  orgs:
+    openshift:
+      repos:
+        ci-tools:
+          require_manually_triggered_jobs: true # enable pipelines on repo level
+```
+
+Secondly, a job that is intended to be promoted to the second stage must be appropriately marked. This involves explicitly setting the `always_run: false` property. Additionally, the job cannot be marked as `optional: true` and should not have `skip_if_only_changed` or `run_if_changed` configured.
+
+```yaml
+- always_run: false # job has to be explicitly marked
+  as: e2e
+  steps:
+    test:
+    - as: e2e
+      commands:
+      ...
+      from: test-bin
+      resources:
+        requests:
+          cpu: 100m
+          memory: 200Mi
+```
+Upon submitting a PR with changes and merging, it may take up to six hours for the changes to be applied. This is because the Test Platform executes a branch protection periodic job every 6h. It updates the GitHub configuration.
+
+## Manual Pipelines
+
+Users can enable manual pipeline support and have complete control over when the second stage is executed. Manual pipelines are recommended for users who:
+- Are comfortable manually executing the second stage of the pipeline.
+- Want to wait for labels and then trigger the second stage of testing, such as trigger manually after receiving an `LGTM` and `approve` label.
+
+Manual pipelines are currently a default solution, so if all tests are properly marked, they should be enabled. To trigger the second stage of the pipeline, user should create a comment on PR `/test remaining-required`. This will run all tests that have been promoted to the second stage.
+
```

Comment:
I think maybe we should add a note here (or somewhere in this doc) that the `/test all` command will trigger the second stage as well

Reply:
hmmm but it won't

Reply:
these tests are "invisible" unless explicitly triggered, then it will

Reply:
oh really? I could have sworn I noticed that happening in `ci-tools`...maybe I am mistaken.